src/lib/rule-finder.js

```diff
@@ -31,9 +31,16 @@ async function _getConfigs(overrideConfigFile, files) {
   return new Set((await Promise.all(configs)).filter(Boolean));
 }
 
+function _getInputArrayOrEmpty(arr) {
+  return arr ? arr : []
+}
+
 async function _getConfig(configFile, files) {
   return Array.from(await _getConfigs(configFile, files)).reduce((prev, item) => {
-    return Object.assign(prev, item, {rules: Object.assign({}, prev.rules, item.rules)});
+    return Object.assign(prev, item, {
+      rules: Object.assign({}, prev.rules, item.rules),
+      plugins: [...new Set([..._getInputArrayOrEmpty(prev.plugins), ..._getInputArrayOrEmpty(item.plugins)])]
```

Comment:
```suggestion
      plugins: [...new Set([].concat(prev.plugins || [], item.plugins || []))]
```
array spread should be avoided when possible.

Reply:
Since `eslint-find-rules` has a minimum of Node@^10.12.0, spread syntax is supported

https://github.com/sarbbottam/eslint-find-rules/blob/bc5a855428f5dc2f8490d16ad5a58dd9665fe110/package.json#L112-L114

Reply:
it's supported (and is still being used in my suggestion), but it should still be avoided most of the time. array spread is super slow, and i find using concat to be much clearer.